### ISSUE TITLE
Accessibility support should be added to WebODF

### ISSUE THREAD

**Issue by peitschie** [kogmbh/WebODF#707]
There is an existing accessibility standard called ARIA that allows web applications to provide hints about what various parts of the DOM do via custom attributes and roles. These are supported to varying degrees by all major browsers, and can greatly improve the accessibility of the site to vision-impaired users.

The official specifications are http://www.w3.org/TR/wai-aria/
